native/CMakeLists.txt

```diff
@@ -2033,8 +1940,8 @@ cocos_source_files(
     cocos/core/utils/IDGenerator.h
     cocos/core/utils/Path.cpp
     cocos/core/utils/Path.h
-    cocos/core/utils/MutableForwardIterator.cpp
-    cocos/core/utils/MutableForwardIterator.h
+    # cocos/core/utils/MutableForwardIterator.cpp
+    # cocos/core/utils/MutableForwardIterator.h
```

Comment:
Why comment it?

Reply:
These two files did not participate in the compilation.